src/realm/error_codes.hpp

```diff
@@ -226,6 +226,12 @@ class ErrorCodes {
         MaintenanceInProgress = RLM_ERR_MAINTENANCE_IN_PROGRESS,
         UserpassTokenInvalid = RLM_ERR_USERPASS_TOKEN_INVALID,
 
+        CallbackFailed = RLM_ERR_CALLBACK,
+        UnknownError = RLM_ERR_UNKNOWN,
+    };
+
+    enum WebSocketError : int32_t {
```

Comment:
Do we want to keep this in the ErrorCodes namespace? Should we just move these into `src/realm/sync/network/websocket.hpp` as a regular error enum? I think error_codes.hpp should be for errors we want users/sdks to have to handle - these seem like more internal status codes specific to the websocket implementation.

Reply:
That sounds like a good idea. Added the change.